mistletoe/contrib/mathjax.py

```diff
@@ -18,11 +18,11 @@ def __init__(self, **kwargs):
 
     def render_math(self, token):
         """
-        Ensure Math tokens are all enclosed in two dollar signs.
+        Convert single dollar sign enclosed expressions to \(\) inline math delimiter for mathjax
```

Comment:
yes this is more thorough. The context here is that mathjax doesn't support single dollar signs because it causes too much fuss as it is too common in normal speech on web pages where matjax is supposed to live. but $ was always inline math and $$ block math from latex syntax, which later added in ``\(\)`` and ``\[ \]`` as alternatives that are easier to parse and read/debug because they are polarized. 

so converting the single dollar signs to the new syntax doesn't represent any change or imposition on the rendering, it is just adjusting standard syntax to be compatible with the modified syntax for mathjax specifically.

Reply:
OK, thanks for the confirmation.